test/end-to-end/core.sh

```diff
@@ -480,6 +509,45 @@ os::cmd::expect_success "oc exec -p ${registry_pod} du /registry > '${LOG_DIR}/p
 
 # make sure there were changes to the registry's storage
 os::cmd::expect_code "diff ${LOG_DIR}/prune-images.before.txt ${LOG_DIR}/prune-images.after.txt" 1
+echo "[INFO] Validated image pruning"
+
+echo "[INFO] Configure registry to accept manifest V2 schema 2"
+os::cmd::expect_success "oc project ${CLUSTER_ADMIN_CONTEXT}"
+os::cmd::expect_success 'oc env -n default dc/docker-registry REGISTRY_MIDDLEWARE_REPOSITORY_OPENSHIFT_ACCEPTSCHEMA2=true'
```

Comment:
No. By default, we want to keep back-wards compatibility with older docker clients. Accepting schema 2 by default means pull-by-id problems for old clients.